docs/topics/mta-web-assigning-application-credentials.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-assigning-application-credentials_{context}"]
+= Assigning application credentials
+
+You can assign credentials to one or more applications.
+
+.Procedure
+
+. In *Developer* view, click *Application inventory*.
+. Click the *Analysis* tab.
++
+// ![](/Tackle2/AddingApps/SelectManageCred.png)
+
+. Click the Options menu {kebab} to the right of *Review* and select *Manage credentials*.
++
+// ![](/Tackle2/AddingApps/ManageCred.png)
+
+. Select a credential from the *Source credentials* list or from the *Maven settings* list.
+//Can you select one of each? more than one of either?
```

Comment:
User can select one of each Source Credentials and/or Maven settings from the list